example/workflows.yml

```diff
@@ -794,6 +794,105 @@ SyncGranule:
       Type: Fail
       Cause: 'Workflow failed'
 
+SyncGranuleCatchWorkflow:
```

Comment:
Just a heads up, I just merged a change where I split up the workflows into separate files so you'll have a merge conflict there.

Reply:
@laurenfrederick Are we using a new parameter `onDuplicateFilename` instead of existing `duplicateHandling` parameter for sync-granule config?

Reply:
@laurenfrederick I can update this work to use the `onDuplicateFilename` field, but I think we still need two workflows. The existing workflow, `SyncGranule`, will not catch the error thrown by the task so that we can test that the workflow fails. The new workflow, `SyncGranuleCatchWorkflow`, will catch the error thrown by the task so we can test that the workflow succeeds.